src/main/java/org/prnhs/javaee/swim/core/CoreConfig.java

```diff
@@ -0,0 +1,14 @@
+package org.prnhs.javaee.swim.core;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@Configuration
+@EnableAutoConfiguration
+@EnableJpaRepositories(basePackages = "org.prnhs.javaee.swim.dao")
+@EntityScan(basePackages = {"org.prnhs.javaee.swim.entity"})
```

Comment:
similar to repositories, we specify where our entities are.  We do this to control where things live and how they are organized.  Make sure we don't end up with spaghetti architecture.